### ISSUE TITLE
Firebase Analytics 21.4.0 is not compatible with the migration to ktx APIs in the main modules

### ISSUE THREAD

**Issue by hnljp** [firebase/firebase-android-sdk#5457]
### [READ] Step 1: Are you in the right place?

I have read what there was to read.

### [REQUIRED] Step 2: Describe your environment

  * Android Studio version: Giraffe 2022.3.1 Patch 2
  * Firebase Component: Analytics
  * Component version: 21.4.0

### [REQUIRED] Step 3: Describe the problem
Also tested with Hedgehog 2023.1.1 RC 1 and Iguana 2023.2.1 Canary 9.
Using Firebase bom 32.4.0

Firebase Analytics 21.4.0 has deprecated the KTX library and the ktx imports, but the kotlin functions are only available using the ktx imports.
All other Firebase components we are using are working with the new imports.

#### Steps to reproduce:

switch from com.google.firebase:firebase-analytics-ktx to com.google.firebase:firebase-analytics:21.4.0
Follow https://firebase.google.com/docs/android/kotlin-migration

Try to use Firebase.analytics without the deprecated ktx import
https://firebase.google.com/docs/analytics/get-started?platform=android#add-sdk

Try to use logEvent without the deprecated ktx import
https://firebase.google.com/docs/analytics/events?platform=android#log_events_3

#### Relevant Code:

try to use:
```
Firebase.analytics
```
and
```
public inline fun @receiver:NonNull FirebaseAnalytics.logEvent(
    name: String,
    block: ParametersBuilder.() -> Unit
): Unit
```

**Comment by hnljp** [comment]
> b/306375392

Thank you very much for your reply.
I am already doing so.
Just wanted to make sure that you where aware of the issue.

**Comment by thatfiredev** [comment]
Hi all,
Thank you for your patience, this has been fixed in [v21.5.0](https://firebase.google.com/support/release-notes/android#analytics_v21-5-0) of the Analytics SDK ([Firebase BoM v32.5.0](https://firebase.google.com/support/release-notes/android#bom_v32-5-0)).

Once you update to those versions, it should be safe to drop the ktx imports and the full package name workaround.

**Comment by hnljp** [comment]
verified, thank you very much.
